man/man8/zfsprops.8

```diff
@@ -843,6 +845,36 @@ is equivalent to
 .Pc .
 .Pp
 The
+.Sy zstd
+compression algorithm provides both high compression ratios and good
+performance. You can specify the
+.Sy zstd
+level by using the value
+.Sy zstd- Ns Em N ,
+where
+.Em N
+is an integer from 1
+.Pq fastest
+to 19
+.Pq best compression ratio .
+.Sy zstd
+is equivalent to
+.Sy zstd-3 .
+You can also specify a negative
+.Sy zstd
+level by using the value
+.Sy zstd-fast- Ns Em N ,
+where
+.Em N
+is an integer from 100
```

Comment:
The commit message for 9530837ff6487a6ceea0fac7e4c674b27885354e says that the valid values for "std-fast are 1-10, 20-100 (inincrements of 10), 500 and 1000". This description doesn't seem to match, unless I'm missing something.

Reply:
Good catch! thank you

Reply:
fixed in e658d9923